### PR TITLE
Treasuries of dexfinance

### PR DESCRIPTION
**Pull request by tuco-gordian**

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/treasury/dexfinance.js: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    Truncated error stack:
    
        at normalizeAddress (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/tokenMapping.js:259:18)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/unwrapLPs.js:882:50
        at Array.map (<anonymous>)
        at sumTokens2 (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/unwrapLPs.js:882:41)
        at sumTokens (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/sumTokens.js:65:12)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/sumTokens.js:49:25
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:29)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:143:15
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:131:29
    Error saving cache: Error: ENOENT: no such file or directory, open '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
        at Object.openSync (node:fs:596:3)
        at Object.writeFileSync (node:fs:2322:35)
        at process.saveCache (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/cache.js:152:17)
        at process.emit (node:events:529:35)
        at process.exit (node:internal/process/per_thread:191:15)
        at process.handleError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/utils/handleError.js:18:11)
        at process.emit (node:events:529:35)
        at emit (node:internal/process/promises:149:20)
        at processPromiseRejections (node:internal/process/promises:283:27)
        at process.processTicksAndRejections (node:internal/process/task_queues:96:32) {
      errno: -2,
      syscall: 'open',
      code: 'ENOENT',
      path: '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
    }

File: projects/treasury/dexfinance.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+			'0x25d887ce7a35172c62febfd67a1856f20faebb00', //pepe
+			'0xf97f4df75117a78c1a5a0dbb814af92458539fb4', //link
+			'0xfc5a1a6eb076a2c7ad06ed22c90d7e710e35ad0a', //gmx
+			'0x912ce59144191c1204e64559fe8253a0e49e6548', //arb
+			'0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f', //wbtc
+		]
+	},
+	avax: {
+		owners: [mainTreasury],
+		ownTokens: [
```
Comment (Define101) [inline projects/treasury/dexfinance.js:30]: replace it for tokens

File: projects/treasury/dexfinance.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+			ADDRESSES.avax.USDT_e,
+			ADDRESSES.avax.WETH_e,
+		]
+	},
+	optimism: {
+		owners: [mainTreasury],
+		ownTokens: [
```
Comment (Define101) [inline projects/treasury/dexfinance.js:37]: replace it for tokens

**Comment by llamatester** [comment]
Error while running adapter at projects/treasury/dexfinance.js: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading 'assets')
    Truncated error stack:
    
        at _call (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/token.js:109:31)
        at async covalentGetTokens (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/token.js:22:20)
        at async Promise.all (index 1)
        at async sumTokens2 (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/unwrapLPs.js:844:22)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:143:9
        at async Promise.all (index 0)
        at async Promise.all (index 0)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:186:3

**Comment by Define101** [comment]
image

**Pull request merged**